cadc-registry/src/main/java/ca/nrc/cadc/reg/client/RegistryClient.java

```diff
@@ -135,46 +142,62 @@ public String getValue() {
     // fully qualified type value (see CapabilitiesReader)
     private static final URI DEFAULT_ITYPE = Standards.INTERFACE_PARAM_HTTP;
 
-    private URL regBaseURL;
-    private String capsDomain;
+    private final List<URL> regBaseURLs = new ArrayList<URL>();
+    // private String capsDomain;
     private boolean isRegOverride = false;
     private int connectionTimeout = 30000; // millis
     private int readTimeout = 60000;       // millis
 
+    /**
+     * Default constructor, using the DEFAULT_CONFIG_FILE_NAME. 
+     * 
+     */
     public RegistryClient() {
-        // standard behaviour: get regBaseURL from config file
-        PropertiesReader propReader = new PropertiesReader(CONFIG_FILE);
+        this(new PropertiesReader(DEFAULT_CONFIG_FILE_NAME));
+    }
+
+    /*
+     * Parameterised constructor to make testing easier.
+     * Needs to be public because the mock tests are in a separate package.
+     * @param configFile The configuration file to use.
+     * 
+     */
+    protected RegistryClient(final File configFile) {
```

Comment:
Looks like the mock tests were moved so that package access should be used. We don't want alternate config file names/locations to be part of the exposed API... and protected is exposed.

Like LocalAuthority: a comment like `// unit tests only` is enough.